src/parser/rogue/assassination/modules/talents/Blindside.tsx

```diff
@@ -24,8 +27,10 @@ class Blindside extends Analyzer {
     enemies: EnemyInstances,
   };
```

Comment:
It's fine here since Enemies isn't used but should probably call it enemyInstances since Enemies is a different module. For sake of clarity.